docs/pages/access-controls/guides/moderated-sessions.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Moderated Sessions
+description: Moderated Sessions
+h1: Moderated Sessions
+---
+
+## Introduction
+
+Moderated Sessions allows Teleport administrators to
+define requirements for other users to be present in a Server or Kubernetes Access session. Depending on the requirements, these users can observe the session in real time, participate in the session, and terminate the session at will.
+
+### Use cases
+
+Moderated Sessions are useful in the following scenarios:
+- When you have stringent security requirements and need to have people watching over user-initiated sessions on a set of servers.
+- When you want to share a terminal with someone else to be able to instruct or collaborate.
+
+## Policies
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who is required to be present to start one.
+
+The system is based around **require policies** and **allow policies**. 
+
+Require policies define a set of conditions that must be a met for a session to start or run.
+A minimum of one policy from each relevant role the user has must match for the session to start.
+
+Allow policies are used to define what sessions a user can join
+and under what conditions they may join a session.
+
+## Configuring Moderated Sessions
+
+### `require_session_join`
+
+#### Options
+
+The following are required options for `require_session_join`:
+
+|Option|Type|Description|
+|---|---|---|
+|`name`|String|The name of the require policy|
+|`filter`|[Filter](./moderated-sessions.mdx#filters)|An expression that, if it evaluates to true for a given user, enables the user to be present in a Moderated Session|
+|`kinds`|`[]`[Session kind](./moderated-sessions.mdx/#session-kinds)|The kind of session that the policy applies to|
+|`modes`|`[]`[Participant mode](./moderated-sessions.mdx#participant-modes)|The participant mode that applies to the user joining the Moderated Session under this policy|
+|`count`|Integer|The number of users that need to match the filter expression to satisfy the policy|
+
+Filter expressions allow for more detailed control over the scope of an allow policy or require policy.
```

Comment:
On second thought, this and the next paragraph make more sense where they originally were, in the "Filters" section. This paragraph is a good example of using filters—which we need in the "Filters" section—and the paragraph beginning "Require policies can specify" only makes sense if you understand the filter syntax, i.e., at the end of the "Filters" section.

However, when we move these paragraphs  back to the "Filters" section, maybe we could use this order of sentences, since the example filter illustrates the `user.roles` and `user.name` fields (we'll need to unescape the code fences).

```markdown
Filter expressions allow for more detailed control over the scope of an allow policy or require policy.

Require policies can specify which users they consider as valid with a filter expression.
The filter context has a `user` object defined with the set fields `roles` and `name`.

Here is an example of a filter expression that evaluates to true if the user is Adam or if the user has the trait `cs-observe`:

\`\`\`
equals(user.name, "adam") || contains(user.roles, "cs-observe")
\`\`\`

```